docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -2236,6 +2236,20 @@ How to **fix** it:
 
 - Increase the allowed limit by using the `-distributor.max-recv-msg-size` option.
 
+### err-mimir-distributor-max-otlp-request-size
+
+This error occurs when a distributor rejects an Otel write request because its message size is larger than the allowed limit after decompression.
+
+How it **works**:
+
+- The distributor implements an upper limit on the message size of incoming OTel write requests after decompression regardless of the compression type. Refer to [OTLP collector compression details](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configgrpc/README.md#client-configuration) for more information.
```

Comment:
Referenced documentation seems irrelevant here, because it points to gRPC client config, but Mimir doesn't accept OTLP/gRPC requests.

I don't think we need to mention "compression" at all in this runbook.

Reply:
nice catch, gracais

Reply:
updated link to confighttp